docs/sources/getting-started/_index.md

```diff
@@ -1,14 +1,141 @@
+- **Promtail** scrapes the log lines from flog, and pushes them to Loki through the gateway.
+- **Grafana** provides visualization of the log lines captured within Loki.
+
+## Prerequisites
+
+- [Docker](https://docs.docker.com/install)
+- [Docker Compose](https://docs.docker.com/compose/install)
+
+## Obtain the test environment
+
+1. Create a directory called `evaluate-loki` for the test environment. Make `evaluate-loki` your current working directory:
+    ```bash
+    mkdir evaluate-loki
+    cd evaluate-loki
+    ```
+1. Download `loki-config.yaml`, `promtail-local-config.yaml`, and `docker-compose.yaml`:
+
+    ```bash
+    wget https://raw.githubusercontent.com/grafana/loki/main/examples/getting-started/loki-config.yaml -O loki-config.yaml
+    wget https://raw.githubusercontent.com/grafana/loki/main/examples/getting-started/promtail-local-config.yaml -O promtail-local-config.yaml
+    wget https://raw.githubusercontent.com/grafana/loki/main/examples/getting-started/docker-compose.yaml -O docker-compose.yaml
+    ```
+
+## Deploy the test environment
+
+All shell commands are issued from the `evaluate-loki` directory.
+
+1. With `evaluate-loki` as the current working directory, deploy the test environment using `docker-compose`:
+    ```bash
+    docker-compose up
+    ```
+1. (Optional) Verify that the Loki cluster is up and running. The read component returns `ready` when you point a web browser at http://localhost:3101/ready. The message `Query Frontend not ready: not ready: number of schedulers this worker is connected to is 0` will show prior to the read component being ready.
+The write component returns `ready` when you point a web browser at http://localhost:3102/ready. The message `Ingester not ready: waiting for 15s after being ready` will show prior to the write component being ready.
+
+## Use Grafana and the test environment
+
+Use [Grafana](https://grafana.com/docs/grafana/latest/) to query and observe the log lines captured in the Loki cluster by navigating a browser to http://localhost:3000.
+The Grafana instance has Loki configured as a [datasource](https://grafana.com/docs/grafana/latest/datasources/loki/).
+
+Click on the Grafana instance's [Explore](https://grafana.com/docs/grafana/latest/explore/) icon to bring up the explore pane.
+
+Use the Explore dropdown menu to choose the Loki datasource and bring up the Loki query browser.
+
+Try some queries.
+Enter your query into the **Log browser** box, and click on the blue **Run query** button.
+
+To see all the log lines that flog has generated:
+```
+{container="evaluate-loki_flog_1"}
+```
+
+The flog app will generate log lines for invented HTTP requests.
+To see all `GET` log lines, enter the query:
+
+```
+{container="evaluate-loki_flog_1"} |= "GET"
+```
+For `POST` methods:
+```
+{container="evaluate-loki_flog_1"} |= "POST"
+```
+
+To see every log line with the value of 401 (unauthorized error): 
+```
+{container="evaluate-loki_flog_1"} |= "401"
```

Comment:
Since this example is using the filter expression as the previous ones, we could add a bit more interesting query:
```suggestion
{container="evaluate-loki_flog_1"} | json | status="401"
```